### PR TITLE
fix issues on login screen

### LINKED ISSUE #153: fix issues on login screen
**Describe the bug**

- [ ] toggle logic is inverted. Should be same as `/register`

- [ ] When user enters incorrect password or email, no information is displayed to him, causing bad UX.

- [ ] Currently there is no option to move back to `/register`. The user has to use back button of the browser. There should be a `link` for the same, for easier navigation.

**To Reproduce**
1. Go to `/login`  
2. 1. Check `/register` where the toggle logic is correct. (Click on eye icon)
    2. Enter wrong details of email or password
    3. There is no option to move back to `/register`. Every web platform has such option for better navigation.

**Expected behavior**

- [ ] Fix toggle logic.

- [ ] Show the information of unsuccessful login to the user.

- [ ] Provide back to `/register` link, similar to _Already have an account? Sign in_ at `/register`

**Screenshots**
login-gif

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04
 - Browser: Chrome
 - Version: 80.0.3987.87

### PR DESCRIPTION
**Pull request by AdityaSrivast**
### Description
- [x] Fix toggle logic.
- [x] Show the information of unsuccessful login to the user.
- [x] Provide back to `/register` link, similar to _Already have an account? Sign in_ at `/register`

Fixes #153 

### Type of Change:
**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)

### Screenshots: 

login-gif1

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] Any dependent changes have been merged

### CODE REVIEW

**Comment by AdityaSrivast** [comment]
@YashMeh @sumedhe please review

File: senz-web/frontend/src/_actions/auth.js [hunk 1]
```diff
@@ -54,6 +54,7 @@ export const LoginAction = ({ email, password }, history) => {
         type: AUTHENTICATION_ERROR,
         payload: "Invalid"
       });
+      alert("Email or password entered is incorrect");
```
Comment (YashMeh) [inline senz-web/frontend/src/_actions/auth.js]: Can we have a component giving this message instead of an alert ?

Comment (AdityaSrivast) [reply senz-web/frontend/src/_actions/auth.js]: @YashMeh on it. Thanks for pointing out.

**Comment by AdityaSrivast** [comment]
@YashMeh please review

Changes made:
- [x] Added component for notification
- [x] Renamed the class to `Login` instead of `Register` in `Login.jsx`

**Comment by AdityaSrivast** [comment]
@YashMeh please review.
Changes made:
- [x] added component for notification
- [x] rename `Register` to `Login` in `Login.jsx`
- [x] modified test in login.test.js

**Pull request merged**